doc/network_methods.rst

```diff
@@ -0,0 +1,16 @@
+######################
+Network Methods
+######################
+
+Network
+-------
+
+.. autoclass:: pypsa.Network
+    :members:
+
+
+Sub-Network
+-----------
+
+.. autoclass:: pypsa.SubNetwork
+    :members:
```

Comment:
Hmm ... we could think about calling this section/page *API reference* and moving it further down in the menu